tools/ci-ts/tests/flux-monitor.test.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from '../test-helpers/common'
 import * as clClient from '../test-helpers/chainlink-cli'
 import { contract, helpers as h, matchers } from '@chainlink/test-helpers'
-import { FluxAggregatorFactory } from '@chainlink/contracts/ethers/v0.6/FluxAggregatorFactory'
+import { FluxAggregatorFactory } from '../../../evm-contracts/ethers/v0.6/FluxAggregatorFactory'
```

Comment:
why was this changed? I think it's cleaner to use the workspace names in imports.

Reply:
As far as I could tell, it wasn't pulling changes I was making to that package because I hadn't published those to `npm`.  This way, the changes can all be part of a single PR, and we can publish as soon as it's accepted.

Reply:
It should incorporate your changes without having to publish anything. The namespaces are automatically resolved. Maybe you were missing a build step? `yarn setup:contracts`